### PR TITLE
Meta Tags Enhancements

### PR DESCRIPTION
**Pull request by mab6bf**
- Adds meta tags to the home page for better previews when linking via Facebook, Twitter, and Slack

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,27 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!--- Facebook Open Graph --->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://sandbox.usaspending.gov/" />
```
Comment (kevinli-work) [inline index.html]: Please fix the URL and anywhere else it references sandbox

File: src/js/helpers/metaTagHelper.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+export const aboutPageMetaTags = {
+    og_url: "https://beta.usaspending.gov/",
+    og_title: "Beta.USAspending.gov",
+    og_description: "Beta.USAspending.gov is the new official source of accessible, searchable and reliable spending data for the U.S. Government.",
+    og_site_name: "Beta.USAspending.gov",
+    og_image: `${productionURL}${imgDirectory}${facebookImage}`,
+    twitter_title: "Beta.USAspending.gov",
+    twitter_description: "Beta.USAspending.gov is the new official source of accessible, searchable and reliable spending data for the U.S. Government.",
+    twitter_image: "https://",
```
Comment (kevinli-work) [inline src/js/helpers/metaTagHelper.js]: All the Twitter images are blank after this.

File: src/js/components/account/Account.jsx [hunk 3]
```diff
@@ -21,6 +24,7 @@ export default class Account extends React.Component {
     render() {
         return (
             <div className="usa-da-account-page">
+                <MetaTags {...MetaTagHelper.federalAccountPageMetaTags} />
```
Comment (kevinli-work) [inline src/js/components/account/Account.jsx]: If we're going to use react-helmet, why not make the titles dynamic (account/award title)?

File: src/js/helpers/metaTagHelper.js (hunk 2 above)
Comment (mab6bf) [inline src/js/helpers/metaTagHelper.js]: Fixed

File: index.html (hunk 1 above)
Comment (mab6bf) [inline index.html]: Fixed

File: src/js/components/account/Account.jsx (hunk 3 above)
Comment (mab6bf) [inline src/js/components/account/Account.jsx]: That functionality should be part of the eventual server-side rendering story

**Pull request merged**